receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper.go

```diff
@@ -371,3 +376,18 @@ func (s *scraper) scrapeAndAppendSignalsPendingMetric(now pcommon.Timestamp, han
 
 	return nil
 }
+
+func (s *scraper) scrapeAndAppendUptimeMetrics(now pcommon.Timestamp, handle processHandle) error {
+	if !s.config.Metrics.ProcessUptime.Enabled {
+		return nil
+	}
+
+	createTime, err := s.getProcessCreateTime(handle)
+	if err != nil {
+		return err
+	}
+	processUptime := now.AsTime().UnixMilli() - createTime
+	s.mb.RecordProcessUptimeDataPoint(now, processUptime)
```

Comment:
it looks like this will be in milliseconds, when your description of the metric says the value is in seconds.